models/AttendanceModel.ts

```diff
@@ -40,8 +40,8 @@ export class AttendanceModel extends BaseEntity {
 
   public getPublicAttendance(): PublicAttendance {
     return {
-      user: this.user?.getPublicProfile(),
-      event: this.event?.getPublicEvent(),
+      user: this.user.getPublicProfile(),
```

Comment:
`PublicAttendance` requires `user` and `event`. Apparently the endpoints for getting attendances for event/current user are broken because `getPublicAttendance` threw.

Reply:
Love that we're catching silly things like this now. 🚀